config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  root to: 'homepages#index'
+
+
+  resources :drivers
+  # don't we need trips :index and :new for the nested routes?
```

Comment:
To answer this question: You might! But in this specific rubric, I'm not looking for it right now ;) But definitely worth considering.